docs/examples/single_screen.rst

```diff
@@ -0,0 +1,316 @@
+************************************
+Single screen synthetic data example
+************************************
+
+This tutorial describes how to generate synthetic data corresponding to a
+single one-dimensional scattering screen. The schematic below shows roughly
+what the screen in this examplewill look like.
+
+.. plot::
+
+    from astropy import units as u
+    import matplotlib.pyplot as plt
+    from screens.visualization import make_sketch
+
+    theta = [-4., -1., 0., 2.] << u.mas
+
+    plt.figure(figsize=(12., 3.))
+    make_sketch(theta)
+    plt.show()
+
+
+Preliminaries
+=============
+
+Start with some standard imports.
+
+.. plot::
+    :include-source:
+    :context:
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import matplotlib.cm as cm
+    from astropy import units as u
+    from astropy import constants as const
+
+Import a colormap to use for phases. Preferably import the colormap from
+the local file ``hue_cycle_cmap``, available for download
+:download:`here <./hue_cycle_cmap.py>`.
+The ``hsv`` colormap is used as fallback.
+
+.. plot::
+    :include-source:
+    :context:
+
+    try:
+        from hue_cycle_cmap import cmap as phasecmap
+    except ImportError:
+        phasecmap = cm.hsv
+
+Also define a function to make two-dimensional phase-intensity colorbars,
+for plotting the electric field.
+
+.. plot::
+    :include-source:
+    :context:
+
+    def phase_intensity_colorbar(fig, ax, phasecmap, ampmax=1.):
+
+        cbar = fig.colorbar(cm.ScalarMappable(), ax=ax, aspect=7.5)
+        cbar_pos = fig.axes[-1].get_position()
+        cbar.remove()
+
+        nph = 36
+        nalpha = 256
+        phases = np.linspace(-np.pi, np.pi, nph, endpoint=False) + np.pi/nph
+        alphas = np.linspace(0., 1., nalpha, endpoint=False) + 0.5/nalpha
+        phasegrid, alphagrid = np.meshgrid(phases, alphas)
+
+        cax = fig.add_axes(cbar_pos)
+        cax.imshow(phasegrid, alpha=alphagrid,
+                   origin='lower', aspect='auto', interpolation='none',
+                   cmap=phasecmap, extent=[-np.pi, np.pi, 0., ampmax])
+        cax.xaxis.tick_top()
+        cax.xaxis.set_label_position('top')
+        cax.yaxis.tick_right()
+        cax.yaxis.set_label_position('right')
+        cax.set_xticks([-np.pi, 0., np.pi])
+        cax.set_xticklabels([r'$-\pi$', '0', r'$\pi$'])
+        cax.set_xlabel('phase (rad)')
+        cax.set_ylabel('normalized intensity')
+
+
+Setting up a scattering screen
+==============================
+
+Set up the screen by defining angles of the scattering points parallel to
+:math:`\theta_\parallel` and perpendicular to :math:`\theta_\perp` the
+direction of effective velocity of the line of sight along the screen.
+For this example, we choose the one-dimensional screen to be perfectly alligned
```

Comment:
I think it is important to phrase this such that it is clear that for a one-dimensional screen, `theta_perp` has to be zero. E.g.,
```
For this example, we want to mimic a one-dimensional screen, 
in which all points appear to be on a line that intersects with the pulsar. 
Hence, we set the perpendicular angles to zero.
```

Reply:
I changed things to your earlier suggestion of leaving out the `theta_perp` altogether. This bit of text is added in the note about using the `dynamic_field` function, where `theta_perp` is used